mcs/class/Makefile

```diff
@@ -39,6 +39,8 @@ build_SUBDIRS :=  \
 	System.Xml.Linq \
 	System.Runtime.InteropServices.RuntimeInformation \
 	System.IO.Compression \
+	System.Drawing			\
+	$(resgen_dir)			\
```

Comment:
Moving resgen into the `build` profile (it needs a dependency here in case tools/ and class/ are built in parallel).